### ISSUE TITLE
Shulkerboxes background

### ISSUE THREAD

**Issue by lazyjesjes** [MC-Synergy/MCSAnalyser-FrontEnd#21]
Add shulkerboxes to the background. As the storage holds more items, more shulkerboxes will appear on the screen. Suppose there fit exactly 100 shulkerboxes on the screen, then for every percent that the storage becomes full of a single type of item, you have a shulkerbox that represents that item. I would also make another shulkerbox for all items that do not yet fit in their own shulkerbox. So, you have that if the storage is 20% full, your screen is also filled with 20% with shulkerboxes. And if the entire storage is full, your screen is also full.
This provides the user with a visual and easy representation of how full the storage is. The user can see at a glance how full the storage is, but in a minecraft jacket.

**Comment by crazyvinvin** [comment]
I love this idea! I think the shulkers boxes would have to go behind the graphs and stay below the navigation bar. This so that it doesn't hurt the user experience too much. Maybe even a button somewhere to disable it, what do you think about this @lazyjesjes?

**Comment by lazyjesjes** [comment]
Yes, a button to disable and enable the shulkerboxes could never hurt. I'd disable it when the site is opened for the first time, so you can enable it to your liking. It might be a bit of an information overload when you open the site the first time. I'd put the button next to the "About" button. In the top-right corner.
